### PR TITLE
[laboratory] make GraphiQL resizable to full viewport size

### PR DESCRIPTION
**Pull request by dimaMachina**
https://github.com/kamilkisiela/graphql-hive/assets/7361780/1518ac11-1b98-4708-8eba-ca5abbd15c25

### CODE REVIEW

File: packages/web/app/src/pages/target-history.tsx [hunk 1]
```diff
@@ -212,7 +212,7 @@ function HistoryPageContent(props: {
       projectId={props.projectId}
       targetId={props.targetId}
       page={Page.History}
-      className="h-full"
+      className="h-[--hive-content-height] overflow-scroll"
```
Comment (dimaMachina) [inline packages/web/app/src/pages/target-history.tsx:215]: scrolls in only content

https://github.com/kamilkisiela/graphql-hive/assets/7361780/edaab028-dbe2-4c19-a130-15454c77f9b7

File: packages/web/app/src/index.css [hunk 2]
```diff
@@ -36,6 +36,8 @@
     --ring: 215 20.2% 65.1%;
 
     --radius: 0.5rem;
+
+    --hive-content-height: calc(100vh - 84px - 47px);
```
Comment (n1ru4l) [inline packages/web/app/src/index.css:40]: Let's have some comments describing these magic values or could we define this variable within the HTML so we don't have to use [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))?

**Review by n1ru4l — APPROVED** [review]
✅ approved with small comment https://github.com/kamilkisiela/graphql-hive/pull/4994/files#r1646149445

File: packages/web/app/src/index.css (hunk 2 above)
Comment (dimaMachina) [reply packages/web/app/src/index.css:40]: added in notion https://www.notion.so/theguildoss/Make-Laboratory-usable-d4d44ccb81b3417db7076d9f572ca12d

Comment (dimaMachina) [reply packages/web/app/src/index.css:40]: added commit in my last PR https://github.com/kamilkisiela/graphql-hive/pull/5000 https://github.com/kamilkisiela/graphql-hive/pull/5000/commits/ebb25852547d27ab9c0a6f40ab1eb59251d39d73

also what do think about naming CSS variables 

- `--header-height`
- `--tabs-navbar-height`
- `--content-height`

Comment (n1ru4l) [reply packages/web/app/src/index.css:40]: nice!

**Pull request merged**